modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-create-heterogeneous-nodepools-agent-hc_{context}"]
+= Creating heterogeneous node pools on agent hosted clusters
+
+You can create heterogeneous node pools on the agent platform. It enables clusters to run diverse machine types, for example, `x86_64` or `ppc64le`, within a single hosted cluster. 
```

Comment:
Do we need to tell users what a heterogeneous node pool is, and why I would use one?